[PROTOTYPE] GUA-245: Copy across solid prototype changes from other repo

huwd opened the pull request:
Prototype Branches are not merged unless they are to become the new default.
This may keep popping up in automated checks for un-merged PRs until we can find a way to ignore `[PROTOTYPE]` or something

An attempt to bring across all changes from here:
https://github.com/alphagov/govuk-account-prototype-ID-save-and-reuse

Should be subject to fairly rigourous checks

huwd committed:
Add SASS classes from other prototype

huwd committed:
Add Data

huwd committed:
add all new views

there are a lot here

henocookie commented:
Some of the layouts and pages aren't displaying as they do in the [govuk-account-prototype-ID-save-and-reuse](https://github.com/alphagov/govuk-account-prototype-ID-save-and-reuse) repo such as the https://govuk-account-id-reuse.herokuapp.com/service/pta/service-pta-home [personal tax page](https://govuk-account-id-reuse.herokuapp.com/service/pta/service-pta-home) so I'll have a proper look before merging.

huwd commented:
Ah ha! interesting, If you drop me a URL I'll hunt down what got lost along the way?

henocookie committed:
remove duplicate CSS to get the prototype working

henocookie committed:
add service name to config file

henocookie commented:
Styles look good now apart from the [unbranded end of task 1 page](https://prototype-preview-gua-245-solicpocurprototype.london.cloudapps.digital/service/dbs/mvp/dbs-end) doesn't look like it does when run locally.

## Running the branch locally 
<img width="1169" alt="Running locally" src="https://user-images.githubusercontent.com/52792328/180772557-163fcf2f-d7fa-4f2d-ac3d-2b8412fe2a20.png">

## [Deployed version of the page](https://prototype-preview-gua-245-solicpocurprototype.london.cloudapps.digital/service/dbs/mvp/dbs-end) 
<img width="1272" alt="Running on deployement" src="https://user-images.githubusercontent.com/52792328/180772570-280d2b07-5d0e-4853-85c8-d774eebea1b0.png">

huwd commented:
Ah interesting, I'll boot that up locally and on the server today and 👀, sounds a bit like it can't get a CSS sheet from somewhere...

henocookie commented:
> Ah interesting, I'll boot that up locally and on the server today and 👀, sounds a bit like it can't get a CSS sheet from somewhere...

Thanks @huwd. I have a feeling because the `unbranded.css` stylesheet is within the public folder, that isn't getting pushed to GitHub. If that's the case and expected behaviour, we'll need to move the styles somewhere that will get pushed 😀

huwd committed:
Lint SASS - account

huwd committed:
Lint SASS - application

huwd committed:
Fix: No such css attribute as `align` + empty rule

Not clear what this was supposed to do, I'm going to assume that this
was supposed to be `text-align` as all the grid alignment css takes
centre / top / bottom and not left

Also remove an empty ruleset

huwd committed:
Add non-prefixed values for compatibility

Working through all the sass warnings to figure out what's wrong

huwd committed:
SASS Lint - explore header

henocookie committed:
add _template file within the DBS MVP folder to style the end of task 1 screen

huwd committed:
Fix Unbranded CSS not compiling

All credit to Dana for figuring this one out. The two stylesheets here
were broken and causing Gulp to error without a good message, bringing
in the base sheets seems to give them what they need

Co-authored-by: Dana Cotoran <dana.cotoran@digital.cabinet-office.gov.uk>
Co-authored-by: Huw Diprose <huw.diprose@digital.cabinet-office.gov.uk>

henocookie committed:
removed temporary layour page under service dvs mvp
